### PR TITLE
Add details on rootless Podman to the readme

### PR DESCRIPTION
**Pull request by mheon**
We didn't really talk about it at all, and it's an exciting feature

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Now is #3033 wanted to add a ptr to this new section, I'd be happy with that.

**Comment by TomSweeneyRedHat** [comment]
LGTM, tests aren't hip.  Good thought to add @mheon.

**Comment by mheon** [comment]
Additional wording suggestions here are welcome - this is likely to be some people's first description of "rootless", so I tried to hit fundamentals, but I might have missed some.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,12 @@ This project tests all builds against each supported version of Fedora, the late
 1. Further work on the podman pod command
 1. Further improvements on rootless containers
 
-## [Shortcomings of Rootless Podman](https://github.com/containers/libpod/blob/master/rootless.md)
+## Rootless
+Podman can be easily run as a normal user, without requiring a setuid binary or any other additional privileges.
+Almost all normal Podman functionality is available, though there are some [shortcomings](https://github.com/containers/libpod/blob/master/rootless.md).
+Any recent Podman release should be able to run rootless without any additional configuration, though your operating system may require some additional configuration detailed in the [install guide](https://github.com/containers/libpod/blob/master/install.md).
+
+## [Shortcomings of Rootless Podman]
```
Comment (karfau) [inline README.md]: Since you linked it above this additional header is not required anymore, right?

Comment (mheon) [reply README.md]: Oh, good catch

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
@baude @TomSweeneyRedHat @rhatdan @vrothberg PTAL

**Pull request merged**